service/lib/agama/storage/proposal.rb

```diff
@@ -186,13 +201,17 @@ def boot_device_issue
       #
       # @return [Issue, nil]
       def missing_devices_issue
-        # At this moment, only the boot device is checked.
-        return unless settings.boot_device
-        return if available_devices.map(&:name).include?(settings.boot_device)
+        available = available_devices.map(&:name)
+        missing = settings.installation_devices.reject { |d| available.include?(d) }
 
-        Issue.new("Selected device is not found in the system",
+        return if missing.none?
+
+        Issue.new(
+          format(_("The following selected devices are not found in the system: %{devices}"),
```

Comment:
Should we use `_n` here?

Reply:
Probably yes because this issue is intended to be reported to the final user.